src/irmin/irmin.mli

```diff
@@ -2209,6 +2220,14 @@ module type S = sig
   type ff_error = [ `No_change | `Rejected | lca_error ]
   (** The type for errors for {!fast_forward}. *)
 
+  type export_error = [ `Disconnected of hash ]
+  (** The type for errors returned by functions that compute closure
+       on the commit graph using the current head. *)
```

Comment:
This seems like an odd description to me, isn't it possible to receive `Error (`Disconnected h)` from `export` by passing an explicit `max` list? (in which case it isn't really to do with the use of a head).

Reply:
Agree, I removed that part as I couldn't find a simple way to put it and it is explained anyway in the doc of `export`.